docs: A sentence does not make sense. (#37230)
angular/angular: aio/content/guide/cli-builder.md

The separation of concerns here is the same as with [schematics](guide/glossary#schematic), which are used for other CLI commands that touch your code (such as `ng generate`).

> * Options are given by the CLI user, context provides access to the CLI Builder API, and the developer provides the behavior.

Reviewer: how about this:

```suggestion
* The `options` object is provided by the CLI user, while the `context` object is provided by the CLI Builder API.
```

Author: Yes, that's much better.

> * The `BuilderContext` object provides access to the scheduling method, `BuilderContext.scheduleTarget()`. The scheduler executes the builder handler function with a given [target configuration](guide/glossary#target).

Reviewer: ```suggestion
* In addition to the contextual information, the `context` object, which is an instance of the `BuilderContext`, also provides access to a scheduling method, `BuilderContext.scheduleTarget()`. The scheduler executes the builder handler function with a given [target configuration](guide/glossary#target).
```

Author: Same here, much clearer.

Author: Since the commit message is not right, I guess someone else will handle these changes. (I'm not sure what to do)

Reviewer 2: @m-ahmadi I see that you created the branch using the github editor. You can download the branch to your local make the changes, add a new commit, and push again. Let me know if you need help with this.
